src/edu/wright/cs/raiderplanner/model/Notification.java

```diff
@@ -81,7 +81,8 @@ public void toggle() {
 	}
 
 	// constructors
-	public Notification(String title, GregorianCalendar dateTime, String details, ModelEntity link) {
+	public Notification(String title, GregorianCalendar dateTime, String details,
+						ModelEntity link) {
```

Comment:
The continuation line should only be indented two additional tabs from the line above.